tgui/packages/tgui/interfaces/NavigationShuttle.tsx

```diff
@@ -248,11 +250,54 @@ export const DisabledScreen = (props) => {
   );
 };
 
+const LaunchHome = (props) => {
+  const { data, act } = useBackend<NavigationProps>();
+
+  return (
+    <Section title="Automatic Return Enabled">
+      <Button.Confirm
+        fluid
+        content={'Return Home'}
```

Comment:
This for whatever reason is visually larger than the other buttons. Also worth noting that the destinations are sharp edged buttons whereas this and the door ones are rounded buttons. The only thing I think needs to change is the font/size should be the same as the destination buttons. Up to you if you also want to standardize the corners too.
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/eb7ce27a-59be-4ff6-9687-503420f0b020)

Reply:
resolved here https://github.com/cmss13-devs/cmss13/pull/5802/commits/9b622e6cf63d0884ac07ee42ae59f51d6cf05a2d